src/Vision/ImageAnnotations.php

```diff
@@ -71,12 +75,15 @@ public function isIdentityDocument(): bool
 
     public function isFrenchNationalIdentityCard(): bool
     {
-        return $this->isIdentityDocument() && \in_array(self::FRENCH_IDENTITY_CARD_LABEL, $this->labels, true);
+        return $this->isIdentityDocument() && (
+            \count(\array_intersect(self::FRENCH_IDENTITY_CARD_LABELS, $this->labels)) > 0 ||
+            \in_array(self::FRENCH_LANGUAGE, $this->webEntities, true)
+        );
     }
 
     public function getFirstNames(): array
     {
-        preg_match('/Prénom( )?\(s\)( )?:( )?(?<first_names>.+)\\n/', $this->text, $matches);
+        preg_match('/Prénom\s?((\(s\))?|s)\s?:\s?(?<first_names>.+)\\n/', $this->text, $matches);
```

Comment:
on my tests, I did not have the `(s)` part on `Prénom(s)` but,  `Prénoms`; So this regex allow this too now